arrow-libs/core/arrow-core-test/src/main/kotlin/arrow/core/test/laws/MonadLaws.kt

```diff
@@ -43,7 +41,6 @@ object MonadLaws {
     M: Monad<F>,
     FF: Functor<F>,
     AP: Apply<F>,
-    SL: Selective<F>,
```

Comment:
These changes are unnecessary since we keep `Selective` around until fully deprecated and removed in `0.13.0`.